### PR TITLE
Containerize Doozer

### PR DESCRIPTION
**Pull request by adammhaile**
@tbielawa @sosiouxme @smunilla @joeldavis84 @vikaslaad 

This is complete, for now.
The remainder of work generally revolves around setting up CI for automatically building this container as well as the new `rundoozer` wrapper. But that will be much easier once this is in master.

Install rundoozer via:
```
pushd ./rundoozer
python setup.py {install,develop} # depending on what you want
popd
```
After that `rundoozer` will automatically pull down the image and run it.
Please run it once first to create `~/.config/rundoozer/settings.yaml` and then open that file and add the ocp_build_data path, your working dir, and anything else you want.

### CODE REVIEW

File: container_install.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+ARCH=$(uname -i)
+
+printf "${GREEN}Installing required yum/yum repos...${NC}\n"
```
Comment (tbielawa) [inline container_install.sh]: Where's this lil `GREEN` buddy comin' from, friend?

Comment (tbielawa) [reply container_install.sh]: Also where is `NC` coming from?

Comment (tbielawa) [reply container_install.sh]: I can't read. This is still in progress.

File: doozer [hunk 2]
```diff
@@ -333,110 +358,6 @@ def images_update_dockerfile(runtime, stream, version, release, repo_type, messa
         yellow_print(msg)
 
 
-@cli.command("images:verify", short_help="Run some smoke tests to verify produced images")
```
Comment (tbielawa) [inline doozer]: RIP :heart:

**Comment by tbielawa** [comment]
Seems legit to me
/lgtm

**Comment by tbielawa** [comment]
Ran into error testing it. We worked this out on Slack. Pasting here for posterity.

```
$ rundoozer --version
It appears you may be using Doozer for the first time.
Be sure to setup Doozer using the user config file:
/home/tbielawa/.config/rundoozer/settings.yaml

Traceback (most recent call last):
  File "/home/tbielawa/.local/bin//rundoozer", line 4, in <module>
    __import__('pkg_resources').run_script('rundoozer==0.3.30', 'rundoozer')
  File "/home/tbielawa/.local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 748, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/home/tbielawa/.local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1524, in run_script
    exec(script_code, namespace, namespace)
  File "/home/tbielawa/.local/lib/python2.7/site-packages/rundoozer-0.3.30-py2.7.egg/EGG-INFO/scripts/rundoozer", line 270, in <module>

  File "/home/tbielawa/.local/lib/python2.7/site-packages/rundoozer-0.3.30-py2.7.egg/EGG-INFO/scripts/rundoozer", line 193, in main

  File "/usr/lib64/python2.7/posixpath.py", line 254, in expanduser
    if not path.startswith('~'):
AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Comment by adammhaile** [comment]
@tbielawa previous error fixed

**Pull request merged**